lib/metrics/names.js

```diff
@@ -272,8 +272,7 @@ const INFINITE_TRACING = {
 const FEATURES = {
   ESM: {
     LOADER: `${SUPPORTABILITY.FEATURES}/ESM/Loader`,
-    UNSUPPORTED_LOADER: `${SUPPORTABILITY.FEATURES}/ESM/UnsupportedLoader`,
-    CUSTOM_INSTRUMENTATION: `${SUPPORTABILITY.FEATURES}/ESM/CustomInstrumentation`
```

Comment:
Same thing about Angler here, should update to remove this metric

Reply:
yep, ill get all these queued up in one PR shortly

Reply:
Here's the PR https://source.datanerd.us/agents/angler/pull/476